src/Models/tests/DetailsModelTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+declare(strict_types=1);
+//namespace Transformers\Models\tests;
```

Comment:
If the namespace isn't necessary, delete it.

Reply:
Removed.